xml/System.Net.Security/SslStream.xml

```diff
@@ -4505,7 +4505,7 @@ This property gets the cipher suite that is going to be used in the communicatio
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- The supported security protocols are specified when the <xref:System.Net.Security.SslStream> is created. If no security protocol was passed to the constructor, the instance is created using <xref:System.Security.Authentication.SslProtocols.Default>. The actual protocol used for authentication is selected based on the protocols supported by the client and server.  
+ The enabled security protocols can be specified in <xref:System.Net.Security.SslStream.AuthenticateAsClient> or via <xref:System.Net.Security.SslClientAuthenticationOptions.EnabledSslProtocols> and <xref:System.Net.Security.SslStream.AuthenticateAsServer> or <xref:System.Net.Security.SslServerAuthenticationOptions.EnabledSslProtocols>. If no security protocol was explicitly specified, the <xref:System.Security.Authentication.SslProtocols.Default> value is used. The actual protocol used for authentication is selected based on the protocols supported by the client and server.  
```

Comment:
```suggestion
 The enabled security protocols can be specified in <xref:System.Net.Security.SslStream.AuthenticateAsClient> or via <xref:System.Net.Security.SslClientAuthenticationOptions.EnabledSslProtocols> and <xref:System.Net.Security.SslStream.AuthenticateAsServer> or <xref:System.Net.Security.SslServerAuthenticationOptions.EnabledSslProtocols>. If no security protocol was explicitly specified, the <xref:System.Security.Authentication.SslProtocols.Default> value is used. The actual protocol used for authentication is selected based on the ones supported by the client and server.  
```

Also I assume the last sentence applies to all cases, whether it be explicitly specified protocols or the default. It might make sense to separate it to its own paragraph. This is only my subjective opinion though, so feel free to disregard.